lib/TestExecutor.py

```diff
@@ -476,6 +476,11 @@ def set_all_noncomplete_case_to_queued(self):
 
     def get_next_queued_case(self):
         xmlnode_next_queued_case = None
+        for each_xmlnode_case in self.list_xmlnode_case:
```

Comment:
Is it necessary? in ccasebase, os.exit() is executed. So regardless how many cases are get queued, it won't execute. So we already get stop on failing case functionality by that os.exit().